nucliadb_models/nucliadb_models/writer.py

```diff
@@ -112,3 +112,9 @@ class ResourceDeleted(BaseModel):
 
 
 ComminResourcePayload = Union[CreateResourcePayload, UpdateResourcePayload]
+
+
+class ResourceFileUploaded(BaseModel):
+    seqid: Optional[int] = None
+    rid: Optional[str] = None
```

Comment:
Some questions that come to my mind:

- Should this be `uuid` instead, to be "coherent" with `ResourceCreated` model?
- Should it be stored in `nucliadb_models`. What policy are we following?

Reply:
Renamed `rid` to `uuid`